geode-membership/src/main/java/org/apache/geode/distributed/internal/membership/gms/MembershipLocatorBuilderImpl.java

```diff
@@ -106,9 +107,13 @@ public MembershipLocatorBuilderImpl(
   @Override
   public MembershipLocator<ID> create()
       throws UnknownHostException, MembershipConfigurationException {
+    TcpSocketCreator sc = this.socketCreator;
+    if (sc == null) {
```

Comment:
Minor point - Can we just default the value of this.socketCreator up above when the variable is initialized, rather than having an if check?

Reply:
I'll do that on my next commit